### PR TITLE
Check in initial versions of the generated Hercules protobufs.

### PR DESCRIPTION
**Pull request by robshakir**
```
 * (A) proto/...
   - Add generated protobufs, along with gen.go files to regenerate
     the .pb.go files.
 * (M) yang/openconfig-hercules-platform-node.yang
 * (M) yang/openconfig-hercules-platform-chassis.yang
   - Correct syntax errors.
```

### CODE REVIEW

File: proto/generate.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Copyright 2017 Google Inc.
```
Comment (wmohsin) [inline proto/generate.sh]: nit: s/2017/2018/  ?

File: proto/openconfig/enums/gen.go [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package enums contains code generated from the Hercules OpenConfig enums
+// protobuf.
+package enums
+
+// pb.go can be generated with go generate.
+//go:generate sh -c "cd $GOPATH/src && protoc --proto_path=. --go_out=:. github.com/openconfig/hercules/proto/openconfig/enums/enums.proto"
```
Comment (wmohsin) [inline proto/openconfig/enums/gen.go]: What is the role of these gen.go files? They all (in various sub-directories) seems to have the same name. And it is unclear why these lines are commented out in each.
(Sorry, I'm not a go person)

File: proto/openconfig/hercules/hercules.proto [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+//   - public/...
+//   - yang/...
+syntax = "proto3";
+
+package openconfig.hercules;
+
+import "github.com/openconfig/ygot/proto/ywrapper/ywrapper.proto";
+import "github.com/openconfig/ygot/proto/yext/yext.proto";
+
+message Hercules {
```
[the remark this answers is not in the document]
Comment (robshakir) [reply proto/openconfig/hercules/hercules.proto]: The module was defined, but doesn't have any contents. I believe the consensus was to leave it there for a placeholder.

File: proto/generate.sh (hunk 1 above)
Comment (robshakir) [reply proto/generate.sh]: Will fix in a follow-up CL.

File: proto/openconfig/enums/gen.go (hunk 2 above)
Comment (robshakir) [reply proto/openconfig/enums/gen.go]: This commented out line allows one to just type "go generate" in each directory and it generates the .pb.go files.

**Pull request merged**